image/system/variants.bzl

```diff
@@ -86,7 +86,6 @@ csp_settings = {
         },
     },
     "qemu": {
-        "autologin": True,
```

Comment:
This doesn't need a `constellation.console` Kernel command line arg?

Reply:
QEMU should provide `hvc0`, which mkosi enables by default now if I'm not mistaken

Reply:
But that alone does not grant autologin, does it?

Reply:
It should, but we'd need to keep the `--autologin` option for that. Or what's your concern?

Reply:
Yes, my concern is that we can't log into QEMU images anymore if we take away the mkosi autologin and add our own based on the cmdline.

Reply:
I think I broke the fact that you could log into any qemu image even stable and nightly images. I will (try) to make this PR have the same behavior as before (i.e. enabling console for all images), though I'm not sure what the original reason was except for we can because it doesn't hurt security on qemu, but now that we have https://github.com/edgelesssys/constellation/tree/main/dev-docs/howto/bare-metal there might be a use-case to also have qemu images where one cannot log in.